documentation/book/con-overview-distributed-tracing.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// assembly-distributed-tracing.adoc
+
+[id='con-overview-distributed-tracing-{context}']
+= Overview of distributed tracing in {ProductName}
+
+Distributed tracing allows developers and system administrators to track the progress of transactions between applications and microservices in a distributed system. This information is useful for monitoring application performance at the micro-level and investigating issues with target systems and end-user applications. In {ProductName}, and data streaming platforms in general, distributed tracing facilitates the end-to-end tracking of messages: from source systems to the Kafka cluster and then to target systems and applications.
+
+As an aspect of system observability, distributed tracing in {ProductName} complements the metrics that are available to view in xref:assembly-metrics-setup-{context}[Grafana dashboards], as well as the available loggers for each component. 
+
+.Introduction to OpenTracing and Jaeger
+
+Distributed tracing in {ProductName} is implemented using the open source {OpenTracingHome} and {JaegerHome} projects.
+
+The OpenTracing specification defines APIs that developers can use to instrument applications -- often in microservice architectures -- for distributed tracing. It is independent from the tracing system.
+
+When instrumented, applications generate __traces__ for individual transactions. Traces are composed of __spans__, which define specific units of work.
+
+To simplify the instrumentation of producers, consumers, and Kafka Streams applications, {ProductName} includes the https://github.com/opentracing-contrib/java-kafka-client/blob/master/README.md[OpenTracing Apache Kafka Client Instrumentation^] library. This library is an example of a __tracer__.
+
+NOTE: The OpenTracing project is merging with the OpenCensus project. The new, combined project is named {OpenTelemetryHome}. OpenTelemetry will provide compatibility for applications that are instrumented using the OpenTracing APIs.
+
+Jaeger, a tracing system, is an implementation of the OpenTracing APIs used for monitoring and troubleshooting microservices-based distributed systems. It consists of four main components and provides client libraries for instrumenting applications. You can use the Jaeger user interface to visualize, query, filter, and analyze trace data.
+
+.An example of a query in the Jaeger user interface
+
+image:image_con-overview-distributed-tracing.png[Simple Jaeger query]
+
+== Distributed tracing support in {ProductName} clients
```

Comment:
Clients would suggets consumers and producers. Which is not the case here.

```suggestion
== Distributed tracing support in {ProductName}
```

Reply:
Updated in the source.